src/main/java/teammates/ui/webapi/UpdateNotificationAction.java

```diff
@@ -18,23 +19,20 @@ public class UpdateNotificationAction extends AdminOnlyAction {
 
     @Override
     public JsonResult execute() throws InvalidHttpRequestBodyException {
-        String notificationId = getNonNullRequestParamValue(Const.ParamsNames.NOTIFICATION_ID);
+        UUID notificationId = UUID.fromString(getNonNullRequestParamValue(Const.ParamsNames.NOTIFICATION_ID));
```

Comment:
I added a method for this, getUUIDRequestParamValue. See GetNotificationAction. Main difference is that string is validated to ensure it is a proper UUID.